core/src/tvu.rs

```diff
@@ -82,7 +82,7 @@ pub struct Tvu {
 }
 
 pub struct TvuSockets {
-    pub fetch: Vec<UdpSocket>,
+    pub fetch: Vec<Arc<UdpSocket>>,
```

Comment:
you do not actually need to change anything here - TVU can be ready to accept shreds across all interfaces at all times